### ISSUE TITLE
Saved checkpoint "fcn_xs"

### ISSUE THREAD

**Issue by Heidisnaps** [apache/incubator-mxnet#8471]
Hello.

I run example/fcn-xs/fcn-xs.py

Where is the trained data stored? 

and 

Is it possible to train my owndata without pre-trained parameters?

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.

**Comment by anirudh2290** [comment]
@Heidisnaps The README now includes additional info on where training data should be stored and where it can be downloaded from https://github.com/apache/incubator-mxnet/blob/master/example/fcn-xs/README.md.  To train your own data you just need to store your data in VOC2012 directory. You can modify the hyperparams used in https://github.com/apache/incubator-mxnet/blob/master/example/fcn-xs/fcn_xs.py. Let us know if you have additional questions.

**Comment by anirudh2290** [comment]
@szha can you please remove "Need Triage" and add "Pending Requester Info" and "Example"

**Comment by safrooze** [comment]
I think this issue has already been addressed above. @yzhliu please close this issue.
